contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -230,8 +230,12 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
             );
         }
 
-        uint256 burnedDifference = BURNED_TX_FEES_ADDRESS.balance - latestBurnedFeesReported;
+        uint256 burnedAddressBalance = BURNED_TX_FEES_ADDRESS.balance;
+        uint256 burnedDifference = burnedAddressBalance - latestBurnedFeesReported;
+        latestBurnedFeesReported = burnedAddressBalance;
```

Comment:
Should this be `latestBurnedFeesReported=burnedDifference`?

Reply:
No, each call of this function should report the incremental change since the last time it was called, so we save the lastest `burnedAddressBalance`. I can rename the variable to `latestBurnedAddressBalance` if that would be more clear.

Reply:
Yeah, renamed.